### ISSUE TITLE
library: empty contructor same line

### ISSUE THREAD

**Issue by Marrikulus** [stefna/codestyle#1]
This is allowed with the current codestyle
```
	public function __construct(
		private readonly AutoPayService $service,
	) {
	}
```
expected correction should be this:
```
	public function __construct(
		private readonly AutoPayService $service,
	) { }
```

**Comment by Marrikulus** [comment]
https://github.com/squizlabs/PHP_CodeSniffer/issues/3291

**Comment by sunkan** [comment]
This have been released in 1.5.0
